### DISCUSSION TITLE
Auto-Merge?

### DISCUSSION THREAD

**Discussion by Steve2955** [importPI19fromDHGE/dhge-pi19-sem6#6] [Ideas]
@RvNovae @ZeroPointMax @nukerxy @photovoltex @sthinbetween @LuisSchaefer 

Wollen wir einen Auto-Merge einführen, wenn alle Bedingungen für einen Merge zutreffen?

![image](https://user-images.githubusercontent.com/21085384/152767593-df9f1e62-91be-4c20-af90-064aa8853a5a.png)

**Comment by RvNovae** [comment]
Hmm da wär ich vorsichtig. Besonders wenn mal mehrere PR in Reihenfolge gemerged werden sollen.

**Reply by emkZero** [reply]
Dies. 

Wir hatten auch seltener mal den Fall, dass irgendein Umstand dann doch trotz Approval einen Merge effektiv blockiert hat.

Oder wenn jemand Umformulierungen mit einem Approval zusammen vorschlägt
